fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java

```diff
@@ -328,6 +328,9 @@ public TPipelineFragmentParams plan(TUniqueId loadId, int fragmentInstanceIdInde
         olapTableSink.init(loadId, taskInfo.getTxnId(), db.getId(), timeout, taskInfo.getSendBatchParallelism(),
                 taskInfo.isLoadToSingleTablet(), taskInfo.isStrictMode(), txnTimeout);
         olapTableSink.setPartialUpdateInfo(uniquekeyUpdateMode, partialUpdateInputColumns);
+        if (uniquekeyUpdateMode != TUniqueKeyUpdateMode.UPSERT) {
```

Comment:
the old StreamLoadPlanner is replaced by NereidsStreamLoadPlanner and will be deleted soon. Is there any unexpected dependence, so that have to do these changes in old code?